packages/DatePicker/src/DatePicker.js

```diff
@@ -244,19 +238,22 @@ function DatePicker(props) {
         hasError={hasError || hasParsingError}
       />
 
-      {shouldShowCalendar ? (
-        <Popover.Content>
-          <div css={calendarPopoverStyles} data-pka-anchor="datepicker.calendar" ref={calendarRef}>
-            <Calendar
-              date={date}
-              isVisible={shouldShowCalendar}
-              onSelect={handleSelect}
-              possibleDate={debouncedPossibleDate}
-              resetPossibleDate={handleResetPossibleDate}
-            />
-          </div>
-        </Popover.Content>
-      ) : null}
+      <Popover.Content>
+        <div
+          css={calendarPopoverStyles}
+          data-pka-anchor="datepicker.calendar"
+          ref={calendarRef}
+          shouldShowCalendar={shouldShowCalendar}
```

Comment:
This file is now reverted back to its original logic pre https://github.com/acl-services/paprika/pull/207.
Only change now is that  I am now passing `shouldShowCalendar` prop instead in order to set a style